docs/Technical-Documentation/jwt-key-rotation.md

```diff
@@ -0,0 +1,70 @@
+# Rotating JWT Keys
+
+## Context
+
+To maintain good security, we will periodically rotate our JWT keys. This document outlines the process of how to do this.
+
+**Warning** Production sites will need to be taken down for maintenance when rotating keys, as the rotation will automatically invalidate all current sessions.
+
+### 1. Generate New Keys
+
+In your Mac terminal (or bash terminal in Windows), enter the following command:
+```bash=
+ssh-keygen -t rsa -b 4096 -m PEM -f jwtRS256.key
+```
+You will receive the following response. Do not add a passphrase when prompted.
+```
+Enter passphrase (empty for no passphrase):
+Enter same passphrase again:
+Your identification has been saved in jwtRS256.key
+Your public key has been saved in jwtRS256.key.pub
+The key fingerprint is:
+# returns key fingerprint
+The key's randmart image is:
+# returns key's randmart image
+```
+Then enter
+```bash=
+openssl rsa -in jwtRS256.key -pubout -outform PEM -out jwtRS256.key.pub
+```
+
+You can now check the contents of your keys with these commands
+```bash=
+cat jwtRS256.key
+# returns private key
+cat jwtRS256.key.pub
+# returns public key
+```
+
+### 2. Base64 Encode Private Key
+
+We use Base64 Encoded Private Keys to make it easier to save to cloud environments and local `.env` files.
+
+```bash=
+openssl enc -base64 -in jwtRS256.key -out jwtRS256.key.base64
```

Comment:
If you don't need to actually write this file we can convert it on the fly with:
```
base64 -w 0 jwtRS256.key.pub
```

Additionally, by using the `-w 0` flag this command will output the entire value on a single line - which can help prevent errors when copying that value over and manually removing the spaces.

Reply:
I thought it was useful to save it just in case of any copy/paste errors we would have a local copy. Good call on the `-w 0` flag.